.gitignore

```diff
@@ -6,3 +6,5 @@ coverage.xml
 pycodestyle.txt
 pylint.txt
 unit-tests.xml
+# editor configurations
```

Comment:
Since `.vscode` is not created by anything in the repo it should be in your global gitignore file and not in the git repo.